amethyst_tiles/src/error.rs

```diff
@@ -0,0 +1,19 @@
+//! Renderer error types.
+
+use amethyst_core::math::{Point3, Vector3};
+
+use err_derive::Error;
+
+/// Tile is out of bounds.
+#[derive(Debug, Error)]
+#[error(
+    display = "Requested coordinate is outside map dimensions: '{:?}', max dimensions=:{:?}",
```

Comment:
Curious, is the `=:{:?}` a convention somewhere? (because earlier you've got single quotes)
Oh ya, if you're changing it, I tend to go with backticks surrounding the debugged item -- copying how Rust does it

Reply:
I agree this looks weird. I was copying it from the previous panic message.

I will update this to be consistantly formatted.

Reply:
Fixed